util/libcrypto.num

```diff
@@ -4527,3 +4527,5 @@ SM2_plaintext_size                      4468	1_1_1	EXIST::FUNCTION:SM2
 conf_ssl_name_find                      4469	1_1_0i	EXIST::FUNCTION:
 conf_ssl_get_cmd                        4470	1_1_0i	EXIST::FUNCTION:
 conf_ssl_get                            4471	1_1_0i	EXIST::FUNCTION:
+err_shelve_state                        4472	1_1_0i	EXIST::FUNCTION:
+err_unshelve_state                      4473	1_1_0i	EXIST::FUNCTION:
```

Comment:
Why are they exported?

Reply:
Not intentionally, but I did see some travis failures, when I did not export them.
On second thought they may be useful even at other places.

Reply:
They are in an internal header file, and only used in libcrypto itself, there really isn't a reason to have them in the .num file.

Reply:
without this hunk I get:
```
/usr/bin/perl util/mkdef.pl crypto linux > libcrypto.map
Error: err_shelve_state does not have a number assigned
Makefile:771: recipe for target 'libcrypto.map' failed
make[1]: *** [libcrypto.map] Error 25
make[1]: Leaving directory '/home/ed/OPC/openssl'
Makefile:170: recipe for target 'all' failed
make: *** [all] Error 2
```
I don't know how to work around that.

Reply:
mkdef.pl reads `include/internal/err.h`, that's why.  Under the circumstances, and considering we might have use for them in libssl as well, I think it's ok to export them, rather than splitting up the declaration of interal err functions in even smaller header files.

Reply:
In general put stuff you don't want to be exported but to be available to all of libcrypto in `crypto/include/internal`

Reply:
I'd prefer not to export unless we need to as a general rule. It is easy to export later if required.

Reply:
So create another header file for these two under `crypto/include/internal`...  yeahok, just make sure to name it so it doesn't clash with files in `include/internal`